examples/r-shiny/README.md

```diff
@@ -0,0 +1,125 @@
+      output$distPlot <- renderPlot({
+        x<- data$SalePrice
+        bins <- seq(min(x), max(x), length.out = input$bins + 1)
+        qplot(x, geom="histogram",breaks =bins,   
+              main = "Histogram for Sale Prices for houses", 
+              xlab = "Sale Price",  
+              fill=I("Orange"), 
+              col=I("darkgray"))        
+    })
+}
+shinyApp(ui = ui, server = server)
+
+```
+Now create another R script which will run above piece of code. Let's name this as `run-app.R`. Start the server using shiny object. Since the host listens on 0.0.0.0, it will be reachable on an appropriate interface address to the connection. The API must use port 8000 for Saturn Cloud to be able to direct traffic to it.
+```R
+shiny::runApp(host="0.0.0.0", port=8000, launch.browser=FALSE)
+```
+
+## Deployment
+
+Save all three files (`setup.R`, `app.R`, `run-app.R`) in a directory inside GitHub repo. Now click **New Deployment**. It can be found on top right side of the resource page. 
+
+![deploy](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/plumber_deployment.png "doc-image")
+
+
+To run, add following to command field.
+```bash
+Rscript run-app.R
+```
+Add path to file in working directory field as shown below. In the screenshot below, my file `run-app.R` is inside repository Dashboard, directory shiny. If you aren't familiar with how to set up SSH credentials and add git repositories to Saturn Cloud check [here](https://saturncloud.io/docs/using-saturn-cloud/gitrepo/).
+
+![deploy](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/Shiny-deploy.png "doc-image")
+
+
+Go to Advanced Settings -> Start Script.  Add following :
+
+`Rscript setup.R`
+
+
+Now `setup.R` will be executed when the deployment is started, which installs shiny and ggplot2. 
+
+![script](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/shiny-sh.png "doc-image")
+
+These are the rest of the fields you'll need to fill out
+
+
+**Name** - the name of the resource, which also is used as the URL for the Dashboard\
+**Instance Count** - How many concurrent machines to host the API on. Set to default as 1.\
+**Hardware and Size** - This example is small hence I have selected smallest available CPU resource.\
+**Image** - set to `saturn-rstudio` image, which is preconfigured to have R and some common R libraries.\
+
+With this you are ready to deploy your dashboard by pressing the green button on the resource page of the deployment. 
+
+## Access Deployed Dashboard
+
+Click the URL given in deployment detail page . You will be redirected to Dashboard page. 
+
+![doc plumber](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/shiny-dashboard.png "doc-image")
+
+Set the slider to the value you want for number of bins for histogram. By default number of bins in this example is set to 15. Histogram for housing sale prices is generated as per the selected value. 
+
+
```

Comment:
Can you add something about publishing the app to everyone on the internet vs the default of just people logged in?